src/Middlewares/PermissionMiddleware.php

```diff
@@ -10,7 +10,7 @@ class PermissionMiddleware
 {
     public function handle($request, Closure $next, $permission)
     {
-        if (Auth::guest()) {
+        if (app('auth')->guest()) {
```

Comment:
Clarification: these changes were primarily to prevent having to enable Facade support in Lumen config, correct?

Reply:
Yep.

Reply:
👍